contribute/editors-checklist.md

```diff
@@ -5,22 +5,23 @@ summary: Checklist for editors before approving and merging a pull request (PR).
 
 ## Before approving and merging a pull request (PR), the editors must check that
 1. The page layout in preview looks correct.
-2. The new page is linked in the appropriate [sidebar](https://github.com/elixir-europe/infectious-diseases-toolkit/tree/main/_data/sidebars) menu, in the same branch of the PR.
-3. The contributors' names are listed in the [CONTRIBUTORS file](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/main/_data/CONTRIBUTORS.yaml), in the same branch of the PR. Advice to have at least one  contributor per page having its contact information in this  [CONTRIBUTORS file](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/main/_data/CONTRIBUTORS.yaml).
-4. All relevant metadata fields in a specific page are correctly filled in (see the [page metadata](/contribute/page-metadata) and the [Editorial board guide](/contribute/editorial-board-guide)). Some critical ones are listed below.
+1. The new page is linked in the appropriate [sidebar](https://github.com/elixir-europe/infectious-diseases-toolkit/tree/main/_data/sidebars) menu, in the same branch of the PR.
+1. The contributors' names are listed in the [CONTRIBUTORS file](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/main/_data/CONTRIBUTORS.yaml), in the same branch of the PR. Advice to have at least one  contributor per page having its contact information in this  [CONTRIBUTORS file](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/main/_data/CONTRIBUTORS.yaml).
+1. All relevant metadata fields in a specific page are correctly filled in (see the [page metadata](/contribute/page-metadata) and the [Editorial board guide](/contribute/editorial-board-guide)). Some critical ones are listed below.
    * unique `page_id` ([List of page IDs](/contribute/website-overview))
    * `contributors`
    * `related_pages` ([Related pages](/contribute/editorial-board-guide.html#related-pages))
    * `training`
    * `search_exclude` must be deleted
    * `description`
-   * `affiliation`
+   * `affiliations` (only used in showcase pages to list countries (2 letter iso-code) and funders)
```

Comment:
```suggestion
   * `affiliations` (only used in showcase pages to list countries (2 letter iso-code) and institutional affiliations of the showcase)
```

Previously discussed in editorial meetings to give priority to countries, institutions, and then projects. It was mentioned that the EU logo is not added here, but the mention for EU funding is added to the acknowledgements section.